core/src/components/cat-input/cat-input.tsx

```diff
@@ -325,7 +325,7 @@ export class CatInput {
                 onFocus={this.onFocus.bind(this)}
                 onBlur={this.onBlur.bind(this)}
                 aria-invalid={this.invalid ? 'true' : undefined}
-                aria-describedby={this.hint?.length ? this.id + '-hint' : undefined}
+                aria-describedby={this.hint?.length || this.invalid ? this.id + '-hint' : undefined}
```

Comment:
This will only solve the issue for inputs, what about all the other form elements?

Reply:
other elements are updated with the same logic (added aria-describedby to those which didn't have it before since hint should support a11y)